lib/ci.js

```diff
@@ -31,7 +31,7 @@ const ci = async () => {
         'later to generate a package-lock.json file, then try again.'
       throw new Error(msg)
     }),
-    rimraf(`${where}/node_modules/`),
+    rimraf(`${where}/node_modules/*`, { glob: { dot: true, nosort: true, silent: true } }),
```

Comment:
does this include `.bin` and other dotfiles?

Reply:
yes! the default options passed to glob from rimraf are `{ nosort: true, silent: true }` adding `dot: true` is the effective change here, which per glob's documentation:

> You can make glob treat dots as normal characters by setting dot:true in the options.